apps/files/models.py

```diff
@@ -131,10 +131,15 @@ def get_mirror(self, addon, attachment=False):
         return posixpath.join(*map(smart_str, [host, addon.id, self.filename]))
 
     def get_url_path(self, src):
+        return self._make_download_url('downloads.file', src)
+
+    def get_signed_url_path(self, src):
+        return self._make_download_url('signing.file', src)
```

Comment:
You could totally use `functools.partial` for this. Or just leave it as it is because that's sensible.